code/game/objects/items/weaponry.dm

```diff
@@ -470,6 +470,15 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 	desc = "A bust of the famous Greek physician Hippocrates of Kos, often referred to as the father of western medicine."
 	icon_state = "hippocratic"
 
+/obj/item/statuebust/attack_self(mob/living/user)
+	. = ..()
+	if(.)
+		return
+	add_fingerprint(user)
+	user.visible_message("[user] rubs some dust off [src].", \

```

Comment:
Statues got a cooldown because this message was spammable, let's not repeat our mistakes.